frontend/src/hooks/common/useToast.ts

```diff
@@ -0,0 +1,22 @@
+import { useEffect, useRef, useState } from 'react';
+
+const useToast = () => {
+  const [isOpen, setIsOpen] = useState(false);
+  const [isAnimating, setIsAnimating] = useState(true);
+  const toastTimer = useRef<NodeJS.Timeout>();
```

Comment:
```suggestion
  const toastTimer = useRef<number | null>(null);
```

브라우저 환경에서 동작하니 number로 가고
undefined보다 null로 없다는걸 명시적으로 하는 건 어떤가요??

아래 setTimeout -> window.setTimeout이 되겠네요

Reply:
얍 수정하였습니다